abjad/tools/schemetools/Scheme.py

```diff
@@ -326,18 +331,20 @@ def format_scheme_value(value, force_quotes=False, verbatim=False):
             else:
                 return value
         elif isinstance(value, bool):
-            if value:
+            if value is True:
                 return '#t'
-            return '#f'
+            else:
```

Comment:
A final return is perfectly explicit. That's the pattern you'll see most of the time in this codebase, especially in recently written or refactored code.